### PR TITLE
Time updates

### PR DESCRIPTION
**Pull request by MaxNoe**
Updates the time definition keywords to how they are defined in the current FITS standard 4.0.

Basically merges `DATE-*` and `TIME-*` into `DATE-*` with a full ISO8601 date and time string

### CODE REVIEW

File: source/general/time.rst [hunk 1]
```diff
@@ -71,15 +71,18 @@ In addition to that main way of specifying times as a floating point number wrt.
...
+    * Observation start date and time
+* ``DATE-AVG`` type: string
+    * Average observation date and time
 * ``DATE-END`` type: string
-    * Observation end date (format: "yyyy-mm-dd")
-* ``TIME-END`` type: string
-    * Observation end time (format: "hh:mm:ss.sss...")
+    * Observation end date and time
 
 Note that the FITS standard allows and it is quite common to instead put a
```
Comment (lmohrmann) [inline source/general/time.rst]: Shouldn't this sentence be updated as well then?

Comment (MaxNoe) [reply source/general/time.rst]: yes

**Comment by lmohrmann** [comment]
Do you still want to await a review from @TarekHC, @MaxNoe? No one has raised objections, so I would merge this now otherwise.

**Comment by TarekHC** [comment]
No, please don't wait for me. Feel free to move on. My availability will be terrible in the next couple of months.

**Pull request merged**